app/src/main/kotlin/com/hippo/ehviewer/coil/CropBorderInterceptor.kt

```diff
@@ -63,10 +63,10 @@ object CropBorderInterceptor : Interceptor {
                     val shouldOverride = r.width() > minWidth && r.height() > minHeight
                     val maybeHwImage = maybeHwBitmap.asCoilImage()
                     val new = if (shouldOverride) {
-                        object : BitmapImageWithRect, Image by maybeHwImage {
-                            override val rect = r
-                            override val image = maybeHwImage
-                        }
+                        BitmapImageWithRect(
+                            rect = r,
+                            image = maybeHwImage,
+                        )
```

Comment:
nit: Make it one-line without named arguments.

Reply:
Done